build.sh

```diff
@@ -3,38 +3,15 @@
 set -euxo pipefail
 
 output_dir=$1
-
-git_commit_hash=${2:-$(git rev-parse HEAD)}
-
-git_version_string=${3:-$(git log --date=format:%Y%m%d --pretty=~git%cd.%h -n 1)}
-
+git_commit_hash=${2:-$(git rev-parse --short HEAD)}
 build_number=${GITHUB_RUN_NUMBER:=0}
 
-ros_distro=${ROS_DISTRO:=foxy}
-
-iname=${PACKAGE_NAME:=px4-msgs}
-
-iversion=${PACKAGE_VERSION:=latest}
-
 docker build \
-  --build-arg UID=$(id -u) \
-  --build-arg GID=$(id -g) \
-  --build-arg ROS_DISTRO=${ros_distro} \
-  --build-arg PACKAGE_NAME=${iname} \
-  --pull \
-  -f Dockerfile.build_env -t "${iname}_build:${iversion}" .
-
-docker run \
-  --rm \
-  -v $(pwd):/${iname}/sources \
-  ${iname}_build:${iversion} \
-  ./packaging/package.sh \
-  -b ${build_number} \
-  -g ${git_commit_hash} \
-  -v ${git_version_string}
+    --build-arg BUILD_NUMBER=${build_number} \
+    -t "ghcr.io/tiiuae/tii-px4-msgs:sha-${git_commit_hash}" .
 
-mkdir -p ${output_dir}
-cp *.deb *.ddeb ${output_dir}
-rm -Rf *.deb *.ddeb
+container_id=$(docker create ghcr.io/tiiuae/tii-px4-msgs:sha-${git_commit_hash})
```

Comment:
This is fine as-is, but if you have time the whole thing can be simplified further, because we're not interested in the built container image (just the .deb artifact from inside of it).

Concretely, we could do:
- Remove the `Dockerfile`
- Have in this `build.sh` script the following command: `$ docker run --rm -it -v "$output_dir:/main_ws" -v "$(pwd):/main_ws/src" ghcr.io/tiiuae/fog-ros-baseimage:builder-3dcb78d /packaging/build.sh -b $build_number` and the result of that command should be that the built artefact is now in `$output_dir`
- Caveat of the above is that changing the baseimage builder reference is not in `Dockerfile` as is the custom in all other repos
- This is just an alternative idea, no need to implement :)

Reply:
I see what you mean but I rather have the explicit Dockerfile and it would look like the same as other similar repos so it is easier to for example add automatic code that would do for us the baseimage updates.
I will leave it as it is for now.

Reply:
Good call, I agree my idea was not perfect :+1: